zerver/models.py

```diff
@@ -1694,6 +1694,7 @@ class RealmAuditLog(models.Model):
     acting_user = models.ForeignKey(UserProfile, null=True, related_name='+', on_delete=CASCADE)  # type: Optional[UserProfile]
     modified_user = models.ForeignKey(UserProfile, null=True, related_name='+', on_delete=CASCADE)  # type: Optional[UserProfile]
     modified_stream = models.ForeignKey(Stream, null=True, on_delete=CASCADE)  # type: Optional[Stream]
+    modified_subscription = models.ForeignKey(Subscription, null=True, on_delete=CASCADE)  # type: Optional[Subscription]
```

Comment:
yeah, we can use the pair as well. In fact I initially did that and then changed my approach to adding a new field so as to have a clarity in things we are doing. Like with this we can clearly say that a subscription toggled with and here is the subscription object affected tied along.